eng/Versions.props

```diff
@@ -1,9 +1,9 @@
 <Project>
   <!--Package versions-->
   <PropertyGroup>
-    <MicrosoftNETSdkPackageVersion>6.0.100-preview.3.21202.5</MicrosoftNETSdkPackageVersion>
-    <MicrosoftAndroidSdkPackageVersion>11.0.200-preview.3.196</MicrosoftAndroidSdkPackageVersion>
+    <MicrosoftDotnetSdkInternalPackageVersion>6.0.100-preview.3.21202.5</MicrosoftDotnetSdkInternalPackageVersion>
+    <MicrosoftAndroidSdkWindowsPackageVersion>11.0.200-preview.3.196</MicrosoftAndroidSdkWindowsPackageVersion>
     <MicrosoftMacCatalystSdkPackageVersion>14.3.100-preview.3.471</MicrosoftMacCatalystSdkPackageVersion>
     <MicrosoftiOSSdkPackageVersion>14.4.100-preview.3.1326</MicrosoftiOSSdkPackageVersion>
```

Comment:
Does the `README.md` file mention how to find these names?

One day they might need to add `$(MicrosoftmacOSSdkPackageVersion)` to get the macOS workload, or maybe the Blazor WebAssembly workload?

Reply:
Curious about the variable naming `MicrosoftAndroidSdkWindowsPackageVersion` ... why Windows?

Reply:
The property name is generated based on the dependency name added to the Version.Details.xml file by my understanding, which needs to correspond to a NuGet package ID. Our Android SDKs have a host OS suffix -- I selected Windows here arbitrarily but all of our packs have the same version so it shouldn't matter which NuGet package ID you use here.